samples/peripheral/802154_phy_test/include/comm_proc.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2019, Nordic Semiconductor ASA
+ * All rights reserved.
+ *
+ * Use in source and binary forms, redistribution in binary form only, with
+ * or without modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions in binary form, except as embedded into a Nordic
+ *    Semiconductor ASA integrated circuit in a product or a software update for
+ *    such product, must reproduce the above copyright notice, this list of
+ *    conditions and the following disclaimer in the documentation and/or other
+ *    materials provided with the distribution.
+ *
+ * 2. Neither the name of Nordic Semiconductor ASA nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * 3. This software, with or without modification, must only be used with a Nordic
+ *    Semiconductor ASA integrated circuit.
+ *
+ * 4. Any software provided in binary form under this license must not be reverse
+ *    engineered, decompiled, modified and/or disassembled.
+ *
+ * THIS SOFTWARE IS PROVIDED BY NORDIC SEMICONDUCTOR ASA "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY, NONINFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NORDIC SEMICONDUCTOR ASA OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+/* Purpose: communication interfaces */
+
+#ifndef COMM_PROC_H__
+#define COMM_PROC_H__
+
+#include <stdint.h>
+
+#include <kernel.h>
+
+/**< Maximum expected size of received packet */
+#define COMM_MAX_TEXT_DATA_SIZE  (320u)
+
+/**< Time to wait for a mark of end of a packet (new line) */
+#define COMM_PER_COMMAND_TIMEOUT (500u)
+
+/**< Maximum size of transmitted packet */
+#define COMM_TX_BUFFER_SIZE      (320u)
+
+/**< Additional symbols in the end of transmitted packet */
+#define COMM_APPENDIX            "\r\n"
+#define COMM_APPENDIX_SIZE       sizeof(COMM_APPENDIX)
+
+/**< Stated of COMM packet parser */
+enum input_state_t {
```

Comment:
General comment - no need for `_t` suffix in enum declaration, it should rather be used with `typedef` if really needed. No need for `_s` in NCS either (it's rather uncommon practice to add any suffixes at all).

Reply:
Removed _t suffixes. Struct _s suffixes temporarily remain, as multiple conflicts would need to be resolved, had the suffixes been removed (some variables would have identical names to a structure name)